### ISSUE TITLE
(Biome Bundle) may have leaked: seen 120 times

### ISSUE THREAD

**Issue by SiennaWesty** [BiomeBundle/BiomeBundle#160]
Does anyone else get this issue on their server?

[08:02:09] [Server thread/DEBUG] [FML]: The world 3d9af9e3 (Biome Bundle) may have leaked: seen 120 times.
 
[08:02:19] [Server thread/DEBUG] [FML]: The world adc9069 (Biome Bundle) may have leaked: seen 60 times.
 
[08:02:24] [Server thread/DEBUG] [FML]: Gathering id map for writing to world save Biome Bundle

**Comment by mccoole** [comment]
Yes, along with a StackOverflowError and a world crash.  Not back to launcher but enough to stop it and potentially loose progress.
